source/integrate-with-integration-environment/choose-the-level-of-authentication.html.md.erb

```diff
@@ -25,16 +25,18 @@ GOV.UK Sign In currently supports the following authentication levels, also know
+    <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">This vector requires your users to have a username and password combination. </span><br><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">You should only use this option if your service does not hold personal information about your users, for example if your service is about booking in an MOT. <br>All services use <code>Cl.Cm</code> as the authentication level by default, unless you change your authentication level to <code>Cl</code>.<br></br>If you request <code>Cl</code>, you will not be able to request identity attributes.</span></td>
   </tr>
   <tr>
     <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">Medium level of protection</span></td>
-    <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent"><code>Cm</code> (credential medium)</span></td>
-    <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">This vector requires your users to have a username and password combination, as well as using 2FA (this will be a one-time password sent through SMS). Additional 2FA methods will be available in the future.</span><br><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">All services use <code>Cm</code> as the authentication level by default, unless you change your authentication level to <code>Cl</code>. </span></td>
+    <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent"><code>Cl.Cm</code> (credential medium)</span></td>
+    <td class="tg-0pky"><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">This vector requires your users to have a username and password combination, as well as using two-factor authentication (2FA). GOV.UK Sign In currently supports 2FA either via a one-time password sent through SMS, or an authenticator app.</span><br><span style="font-weight:400;font-style:normal;text-decoration:none;color:#000;background-color:transparent">All services use <code>Cl.Cm</code> as the authentication level by default, unless you change your authentication level to <code>Cl</code>. <br></br>If you need to request identity attributes, you must request <code>Cl.Cm</code>. </span></td>
   </tr>
 </tbody>
 </table>
 
+You'll include your level of authentication in your authorisation request.
```

Comment:
```suggestion
Include your level of authentication in your authorisation request.
```

Reply:
going to leave this as is, because they're not yet making their auth request (this comes later but users were wondering when they include their level of authentication)